galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,161 @@
+                    }],
+                    'shape': {
+                        'title': 'Sync Red Hat certified collections',
+                        'description':
+                            'Configure access to sync collections to Private Automation Hub.',
+                        'link': {
+                            'title': 'Get started',
+                            'href': './ansible/automation-hub/token',
+                        },
+                    },
+                }],
+                'try': [
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Install Private Automation Hub',
+                            'link': {
+                                'title': 'Get started',
+                                'external': True,
+                                'href': 'https://access.redhat.com/documentation/en-us/\
+                                    red_hat_ansible_automation_platform/1.2/html/\
+                                    installing_and_upgrading_private_automation_hub/index',
+                            },
+                        },
+                    },
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Manage repositories in Private Automation Hub',
+                            'description': 'Add community and privately developed collections \
+                                to your Private Automation Hub.',
+                            'link': {
+                                'title': 'Get started',
+                                'external': True,
+                                'href': 'https://access.redhat.com/documentation/en-us/\
+                                    red_hat_ansible_automation_platform/1.2/html/\
+                                    publishing_proprietary_content_collections_in_\
+                                    automation_hub/index',
+                            },
+                        },
+                    },
+                ],
+                'learn': [
+                    {
+                        'permissions': [{
+                            'method': 'isEntitled',
+                            'args': ['ansible'],
+                        }],
+                        'shape': {
+                            'title': 'Connect Automation Hub to your automation infrastructure',
+                            'link': {
+                                'title': 'Get started',
+                                'external': True,
+                                'href': 'https://docs.ansible.com/ansible-tower/latest/html/userguide/\
```

Comment:
I'm not sure `\` works correctly for escaping new lines here. I think you'll end up with all the spaces in the url.

Reply:
I gonna rest it today, i had the same thoughts, thanks for the reminder

Reply:
Fixed, thanks